src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -215,11 +213,21 @@ std::string postQueryContext
     return answer;
   }
 
-  s = xmlTreat(cleanPayload, ciP, &parseData, RtQueryContextResponse, "queryContextResponse", NULL, &errorMsg);
+  //
+  // NOTE
```

Comment:
I haven't needed this before, this is the first time.
However, all GET conv-op (that translate into a ngsi10 query) will get to this std-op, so, this unique comment covers them all ...

If the same thing is needed for some other "conv-op translation to std-op", I will try to add the same 'NOTE'.